src/modules/jcms_notifications/src/Notification/SnsMessage.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\jcms_notifications\Notification;
+
+final class SnsMessage {
```

Comment:
this isn't really tied to SNS, but has important logic related to the JSON format of the message. Could it be `BusMessage` or `BusOutgoingMessage`?